src/layers/domain/core/aggregate_root.py

```diff
@@ -0,0 +1,17 @@
+from pydantic import BaseModel, Field
+
+from .event import Event
+
+
+class AggregateRoot(BaseModel):
+    class Config:
+        arbitrary_types_allowed = True
+
+    events: list[Event] = Field(default=(lambda: [])())
```

Comment:
```suggestion
    events: list[Event] = Field(default_factory=list)
```

Reply:
Ah, yes.  Too used to typed languages where the type and constructor/function are two different things.